modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -272,9 +271,7 @@
           reducible))))))
 
 (defmethod sql-jdbc.sync/syncable-schemas :redshift
-  [driver conn metadata]
+  [_ conn metadata schema-inclusion-patterns schema-exclusion-patterns]
   (reducible-schemas-with-usage-permissions
    conn
-   (eduction
-    (remove (set (sql-jdbc.sync/excluded-schemas driver)))
-    (sync.describe-database/all-schemas metadata))))
+   (sql-jdbc.sync/syncable-schemas :sql-jdbc conn metadata schema-inclusion-patterns schema-exclusion-patterns)))
```

Comment:
```suggestion
   ((get-method sql-jdbc.sync/syncable-schemas :sql-jdbc) driver conn metadata schema-inclusion-patterns schema-exclusion-patterns)))
```

Reply:
You **definitely** should be invoking "parent methods" this way. Why? Because if `syncable-schemas` in turn calls some **other** multimethod, it's going to use the `driver` parameter you passed it for dispatch. If you pass it `:sql-jdbc`, then it's using the `:sql-jdbc` version of any methods it calls, even if there's a `:redshift`-specific override. So best practice here is to manually get the `:sql-jdbc` version of the method and then pass in the `driver` parameter you got.

Why not pass in `:redshift` directly instead of `driver`? Well, we don't know that someone isn't extending the Redshift driver in some way. This method might actually be getting invoked for some `:redshift-2` driver or something like that. So it's important to pass the actual `driver` we got, not what we think it is. (A good example of this would be the `:postgres` driver. The Postgres driver has to be explicit to pass around `driver` rather than `:postgres`, or else it would skip over Redshift's implementations of things)

Reply:
Thanks, learned about this in the past but somehow forgot this time.